src/components/organisms/Footer/Footer.tsx

```diff
@@ -0,0 +1,46 @@
+import {
+  IconBrandTelegram,
+  IconBrandFacebook,
+  IconBrandInstagram,
+  IconBrandYoutube,
+} from '@tabler/icons-react';
+import Link from 'next/link';
+
+import CircleLink from '@/components/atoms/CircleLink/CircleLink';
+import { CONTACTS_URL } from '@/constants/routes';
+
+import styles from './Footer.module.scss';
+
+export default function Footer() {
+  const currentYear = new Date().getFullYear();
+
+  return (
+    <div className={styles.footer}>
+      <div className={styles.socialLinks}>
+        <CircleLink
+          link="https://www.facebook.com/savitskymuseum/"
+          icon={<IconBrandTelegram color="white" size="20" />}
+        />
+        <CircleLink
+          link="https://www.instagram.com/museumsavitsky1/"
+          icon={<IconBrandInstagram color="white" size="20" />}
+        />
+        <CircleLink
+          link="https://t.me/museumsavitsky"
+          icon={<IconBrandFacebook color="white" size="20" />}
+        />
+        <CircleLink
+          link="https://www.youtube.com/channel/UCshZAh9gVGMv_gFQJo3IXyQ"
+          icon={<IconBrandYoutube color="white" size="19" />}
+        />
+      </div>
+      <div className={styles.copyright}>
+        ©{currentYear}. All copyrights reserved. Copying website content allowed only on permission
```

Comment:
1. How about using `&copy;` HTML meta-symbol to guarantee that browser interprets symbol properly?
2. Let's, please, put these messages (including contact details) into translation messages so it could translated later easily without affecting source code